api/rosetta/transaction_integration_test.go

```diff
@@ -72,23 +69,18 @@ func TestAPI_Transaction(t *testing.T) {
 		{
 			name:       "some cherry picked transaction",
 			request:    requestTransaction(firstHeader, firstTx),
-			validateTx: validateTransfer(t, firstTx, "754aed9de6197641", "631e88ae7f1d7c20", 1),
```

Comment:
This test case was removed since it's the new scenario with account creation that has multiple deposits and withdrawals in a single transaction, while we're typically verifying a simple transfer.

The downside is that we no longer have a test case where we're verifying both transactions from a single block - we're only verifying the second one.